README.md

```diff
@@ -1,2 +1,70 @@
-# .github
-Organization-wide repository for common governance documents.
+# Notary Project Overview
+
+The Notary Project is a set of specifications and tools intended to provide a cross-industry standard for securing software supply chains. 
+
+The Notary Project started with an implementation for signing images in container registries and ensuring their integrity before deployment. The initial implementation uses [The Update Framework (TUF)](https://github.com/theupdateframework) and requires registries to host additional server infrastructure for managing signing keys and image metadata. This server infrstrcture tightly integrates with the container registry and keeps track of the images pushed to the registry. There is also a client component in the form of the `notary` command line interface (CLI) that can be used by developers or CI/CD pipelines to sign and push container images and update the metadata. The CLI wrapps the communication to the registry as well as to the key and metadata management server component. The most prominent use of this implementation is in Docker Content Trust (DCT). The server and the client implementation can be found in the [notary](https://github.com/notaryproject/notary) repository under the Notary Project organization.
```

Comment:
typo: 

"This server `infrstrcture` tightly" ==> "This server `infrastructure` tightly"

Reply:
third sentence from the end "The CLI wrapps" ==> "The CLI wraps"